Werkzeug updated. Getting errors

On Werkzeug 2.0, getting error: 

`[ERROR] Runtime.ImportModuleError: Unable to import module 'wsgi_handler': No module named 'werkzeug._compat' Traceback (most recent call last):`

I running Python3 (Flask 2.0.0) and still getting this error. 
This is the error that I am getting:

>```[ERROR] ModuleNotFoundError: No module named 'werkzeug._compat' Traceback (most recent call last):   File "/var/task/serverless_sdk/__init__.py", line 144, in wrapped_handler     return user_handler(event, context)   File "/var/task/s_api.py", line 25, in error_handler     raise e   File "/var/task/s_api.py", line 20, in <module>     user_handler = serverless_sdk.get_user_handler('wsgi_handler.handler')   File "/var/task/serverless_sdk/__init__.py", line 56, in get_user_handler     user_module = import_module(user_module_name)   File "/var/lang/lib/python3.8/importlib/__init__.py", line 127, in import_module     return _bootstrap._gcd_import(name[level:], package, level)   File "<frozen importlib._bootstrap>", line 1014, in _gcd_import   File "<frozen importlib._bootstrap>", line 991, in _find_and_load   File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked   File "<frozen importlib._bootstrap>", line 671, in _load_unlocked   File "<frozen importlib._bootstrap_external>", line 783, in exec_module   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed   File "/var/task/wsgi_handler.py", line 22, in <module>     import serverless_wsgi   File "/var/task/serverless_wsgi.py", line 18, in <module>     from werkzeug._compat import BytesIO, string_types, to_bytes, wsgi_encoding_dance```

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,6 +1,5 @@
 language: python
 python:
-  - "2.7"
   - "3.6"
   - "3.7"
   - "3.8"

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,1 +1,1 @@
-werkzeug==1.0.1
+werkzeug>2

--- a/serverless_wsgi.py
+++ b/serverless_wsgi.py
@@ -8,6 +8,7 @@ Inspired by: https://github.com/miserlou/zappa
 Author: Logan Raarup <logan@logan.dk>
 """
 import base64
+import io
 import json
 import os
 import sys
@@ -15,7 +16,7 @@ from werkzeug.datastructures import Headers, iter_multi_items, MultiDict
 from werkzeug.wrappers import Response
 from werkzeug.urls import url_encode, url_unquote, url_unquote_plus
 from werkzeug.http import HTTP_STATUS_CODES
-from werkzeug._compat import BytesIO, string_types, to_bytes, wsgi_encoding_dance
+
 
 # List of MIME types that should not be base64 encoded. MIME types within `text/*`
 # are included by default.
@@ -26,6 +27,7 @@ TEXT_MIME_TYPES = [
     "application/vnd.api+json",
     "image/svg+xml",
 ]
+_STRING_TYPES = (str,)
 
 
 def all_casings(input_string):
@@ -115,15 +117,15 @@ def get_script_name(headers, request_context):
 def get_body_bytes(event, body):
     if event.get("isBase64Encoded", False):
         body = base64.b64decode(body)
-    if isinstance(body, string_types):
-        body = to_bytes(body, charset="utf-8")
+    if isinstance(body, _STRING_TYPES):
+        body = body.encode('utf-8')
     return body
 
 
 def setup_environ_items(environ, headers):
     for key, value in environ.items():
-        if isinstance(value, string_types):
-            environ[key] = wsgi_encoding_dance(value)
+        if isinstance(value, _STRING_TYPES):
+            environ[key] = value.encode('utf-8').decode('latin1', 'replace')
 
     for key, value in headers.items():
         key = "HTTP_" + key.upper().replace("-", "_")
@@ -214,7 +216,7 @@ def handle_payload_v1(app, event, context):
         "SERVER_PORT": headers.get(u"X-Forwarded-Port", "80"),
         "SERVER_PROTOCOL": "HTTP/1.1",
         "wsgi.errors": sys.stderr,
-        "wsgi.input": BytesIO(body),
+        "wsgi.input": io.BytesIO(body),
         "wsgi.multiprocess": False,
         "wsgi.multithread": False,
         "wsgi.run_once": False,
@@ -274,7 +276,7 @@ def handle_payload_v2(app, event, context):
         "SERVER_PORT": headers.get(u"X-Forwarded-Port", "80"),
         "SERVER_PROTOCOL": "HTTP/1.1",
         "wsgi.errors": sys.stderr,
-        "wsgi.input": BytesIO(body),
+        "wsgi.input": io.BytesIO(body),
         "wsgi.multiprocess": False,
         "wsgi.multithread": False,
         "wsgi.run_once": False,
@@ -332,7 +334,7 @@ def handle_lambda_integration(app, event, context):
         "SERVER_PORT": headers.get(u"X-Forwarded-Port", "80"),
         "SERVER_PROTOCOL": "HTTP/1.1",
         "wsgi.errors": sys.stderr,
-        "wsgi.input": BytesIO(body),
+        "wsgi.input": io.BytesIO(body),
         "wsgi.multiprocess": False,
         "wsgi.multithread": False,
         "wsgi.run_once": False,

--- a/setup.py
+++ b/setup.py
@@ -5,7 +5,7 @@ with open("README.md", "r") as fh:
 
 setuptools.setup(
     name="serverless-wsgi",
-    version="1.7.8",
+    version="2.0.0",
     author="Logan Raarup",
     author_email="logan@logan.dk",
     description="Amazon AWS API Gateway WSGI wrapper",
@@ -13,7 +13,7 @@ setuptools.setup(
     long_description_content_type="text/markdown",
     url="https://github.com/logandk/serverless-wsgi",
     py_modules=["serverless_wsgi"],
-    install_requires=["werkzeug==1.0.1"],
+    install_requires=["werkzeug>2"],
     classifiers=(
         "Development Status :: 5 - Production/Stable",
         "Programming Language :: Python :: 2",

